BML_TUX_Project/Packages/bmlTUX/Scripts/UI/RuntimeUI/ExperimentGui.cs

```diff
@@ -119,7 +119,8 @@ public void StartExperimentFromButton()
 
         [PublicAPI]
         public void StartDebugExperimentFromButton() {
-            Session session = new DebugSession();
+            Session session = new DebugSession(runner.DesignFile.GetFileLocationSettings.DebugFolder,
+                runner.DesignFile.GetFileLocationSettings.DebugFileName);
```

Comment:
The fact that both parameters need to reference `runner.DesignFile.GetFileLocationSettings`, suggests to me that we might want to pass an object around here instead or think of a way to clean this up a bit.